server/test/data/noncommercial-model-test-data.es6

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+var factory = require('unionized');
+
+module.exports = {
+  noncommercialModelPerson: factory.factory(
+    {
+      applicationId: 1,
+      appControlNumber: '',
+      controlNumber: undefined,
+      region: '11',
+      forest: '11',
+      district: '11',
+      authorizingOfficerName: 'Test Name',
+      authorizingOfficerTitle: 'Test Title',
+      eventName: 'Test Event',
+      applicantInfoPrimaryFirstName: 'Joe',
+      applicantInfoPrimaryLastName: 'Smith',
+      applicantInfoDayPhoneAreaCode: '555',
```

Comment:
Can we make the phone number chunks different values? An earlier typo wasn't caught by tests because the values for area code and prefix were the same.